src/components/CoverPhoto/index.tsx

```diff
@@ -72,13 +74,13 @@ export default class CoverPhoto extends React.Component<CoverPhotoProps> {
             className={css`
               margin: 0 0 0.8rem;
               font-size: 3rem;
-              max-width: 460px;
             `}
           >
             {this.props.headline}
           </h1>
           <Line />
           <Byline authors={this.props.authors} />
+          <p>{this.props.explainer}</p>
```

Comment:
```suggestion
          {!!this.props.explainer && <p>{this.props.explainer}</p>}
```
so we don't render the p tag when there is not explainer